nucypher/cli/painting.py

```diff
@@ -224,11 +224,13 @@ def paint_staged_stake(emitter,
 Staking address: {staking_address}
 ~ Chain      -> ID # {stakeholder.blockchain.client.chain_id} | {stakeholder.blockchain.client.chain_name}
```

Comment:
There is not `stakeholder.blockchain` anymore, so this f-string fails https://github.com/nucypher/nucypher/issues/1265